### ISSUE TITLE
Show summary statistics on selected files e.g. show total file size for all selected files

### ISSUE THREAD

**Issue by xiaodaigh** [fman-community/fman#292]

**Comment by mherrmann** [comment]
Thank you for the suggestion. Where would you like to see this information? In eg. the Size column, or in the status bar at the bottom?

**Comment by xiaodaigh** [comment]
Windows has it at the bottom. But i guess i am open to better placement as
well

**Comment by joshas** [comment]
There's plugin for this: https://github.com/kek91/StatusBarExtended
Although I wouldn't mind if this functionality would be implemented in fman core. Take a look at TC status display - pretty simple and informative.
![image](https://user-images.githubusercontent.com/79141/35297898-1925bd4a-0089-11e8-959a-30e3ec2e2527.png)

**Comment by mherrmann** [comment]
Yes, this would be nice. StatusBarExtended is a great first solution, but not yet perfect because fman does not expose all necessary APIs. In particular, StatusBarExtended cannot notice when you switch to the other pane (#68). Further, ideally there should probably be _two_ status bars, one where you see information for the left pane and another for the right one.

**Comment by mherrmann** [comment]
As pointed out in #418, the number of selected files should also be displayed.

**Comment by diovon** [comment]
I'm missing this feature too. Now when I select few files I just see the word ready in the status bar and it would be nice to see how much files I selected and maybe the sum up size of them.
